source/common/ssl/tls_certificate_config_impl.cc

```diff
@@ -0,0 +1,19 @@
+#include "common/ssl/tls_certificate_config_impl.h"
+
+#include "common/config/datasource.h"
+
+namespace Envoy {
+namespace Ssl {
+
+TlsCertificateConfigImpl::TlsCertificateConfigImpl(const envoy::api::v2::auth::Secret& config)
+    : message_([&config] {
+        envoy::api::v2::auth::Secret message;
+        message.CopyFrom(config);
+        return message;
+      }()),
```

Comment:
Line 9 to 13 is overkill.    I think it can be simplified as:   message_(config)

Reply:
Per my other comment you can remove all of this entirely, I don't think it's needed.

Reply:
Replaced